reddit.js

```diff
@@ -120,6 +120,13 @@
     });
   };
 
+  reddit.user = function (username, where = "") {
```

Comment:
There's an `aboutUser` resource [defined](https://github.com/sahilm/reddit.js/blob/master/reddit.js#L117-L121). I think that should be replaced with this new resource.